solana/program_v2/packages/client/cli/package.json

```diff
@@ -13,35 +13,16 @@
   "files": [
     "/bin",
     "/dist",
-    "/npm-shrinkwrap.json",
     "/oclif.manifest.json"
   ],
   "dependencies": {
-    "@identity.com/gateway-solana-client": "^2.0.0-beta.8",
-    "@project-serum/anchor": "^0.26.0",
-    "@project-serum/anchor-cli": "^0.26.0",
-    "@oclif/core": "^1",
-    "@oclif/plugin-help": "^5",
-    "@oclif/plugin-plugins": "^2.0.1",
-    "@solana/web3.js": "^1.50.1"
+    "@identity.com/gateway-solana-client": "^2.0.0-beta.12"
   },
   "devDependencies": {
-    "@oclif/test": "^2",
-    "@types/chai": "^4",
-    "@types/fs-extra": "^9.0.13",
-    "@types/mocha": "^9.0.0",
-    "@types/node": "^16.9.4",
-    "chai": "^4",
-    "eslint": "^7.32.0",
-    "eslint-config-oclif": "^4",
-    "eslint-config-oclif-typescript": "^1.0.2",
-    "globby": "^11",
-    "mocha": "^9",
     "oclif": "^3",
-    "shx": "^0.3.3",
-    "ts-mocha": "^10.0.0",
-    "ts-node": "^10.2.1",
-    "tslib": "^2.3.1",
+    "@oclif/core": "^1",
```

Comment:
That's directly used in `add-gatekeeper.ts` and therefore I think it should be a `dependency`. (But double check with the oclif framework)